huaweicloud/services/iam/resource_huaweicloud_identity_project.go

```diff
@@ -134,7 +135,17 @@ func resourceIdentityProjectV3Delete(_ context.Context, d *schema.ResourceData,
 
 	err = projects.Delete(identityClient, d.Id()).ExtractErr()
 	if err != nil {
-		return fmtp.DiagErrorf("Error deleting Huaweicloud project: %s", err)
+		if _, ok := err.(golangsdk.ErrDefault404); ok {
+			errorMsg := "Deleting projects is not supported, and it will return to success temporarily."
```

Comment:
suggest to warn that: it only remove projects from the state but remains on the cloud.